alibi_detect/saving/_tensorflow/loading.py

```diff
@@ -21,6 +21,7 @@
 from alibi_detect.models.tensorflow.autoencoder import (AE, AEGMM, VAE, VAEGMM,
                                                         DecoderLSTM,
                                                         EncoderLSTM, Seq2Seq)
+from alibi_detect.utils.tensorflow.misc import check_model
```

Comment:
Minor, but why not keep the function in this module since it's specifically used during loading?

Reply:
Mmn I put it here so that its next to `clone_model`, which is quite related. But can move back to `saving` since we only use it there...